### PR TITLE
/E10/

### PR DESCRIPTION
**Pull request by viridispirus**

### CODE REVIEW

**Comment by viridispirus** [comment]
Ops... sorry I don't wanted to push all other commits again. Please told me what I maked wrong. And I avoid to do it again.

File: doc/Requirements Specification/Parts/Environment.lyx [hunk 1]
```diff
@@ -109,7 +109,7 @@ name "/E10/"
 
 \end_inset
 
- Beagle requires Java and Eclipse to run.
+ Beagle requires not less than Java 7 and Eclipse Mars to run.
```
Comment (flyfl) [inline doc/Requirements Specification/Parts/Environment.lyx]: Beagle should run on a Java 7 runtime environment (or higher) and on an Eclipse distribution that is up to date with Eclipse Mars (4.5).

**Comment by flyfl** [comment]
I think that lots of your points belong into "funtional requirements", again.

**Comment by viridispirus** [comment]
Why it should belong to "funtional requirements"? This is in my opinion something that describes the software environment. And belong to "Environment".

**Comment by RomanLangrehr** [comment]
> I think that lots of your points belong into "funtional requirements", again.

I have to agree, I think /OQ30/, /OQ50/ and /OQ60/ are optional functional requirements.

**Comment by viridispirus** [comment]
Ok, I will change that.

**Comment by RomanLangrehr** [comment]
Please merge #82 first. This PR contains the commits of #82

**Comment by jGleitz** [comment]
> Please merge #82 first. This PR contains the commits of #82

Not anymore

File: doc/Requirements Specification/Parts/Environment.lyx [hunk 2]
```diff
@@ -109,7 +109,8 @@ name "/E10/"
 
 \end_inset
 
- Beagle requires Java and Eclipse to run.
+ Beagle should run on Java 7 runtime environment (or higher) and Eclipse
```
Comment (jGleitz) [inline doc/Requirements Specification/Parts/Environment.lyx]: Please specify Java 8!

**Comment by flyfl** [comment]
lgtm

**Comment by RomanLangrehr** [comment]
Lgtm

**Pull request merged**